build.sbt

```diff
@@ -57,29 +57,28 @@ val googleApiServicesStorageVersion = s"v1-rev20211201-$googleClientsVersion"
 
 // check versions from libraries-bom
 // https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/index.html
-val animalSnifferAnnotationsVersion = "1.20"
-val bigQueryStorageBetaVersion = "0.134.0"
-val bigQueryStorageVersion = "2.10.0"
-val checkerFrameworkVersion = "3.21.2"
-val errorProneAnnotationsVersion = "2.11.0"
+val animalSnifferAnnotationsVersion = "1.21"
+val bigQueryStorageBetaVersion = "0.136.2"
+val bigQueryStorageVersion = "2.12.2"
+val checkerFrameworkVersion = "3.21.4"
+val errorProneAnnotationsVersion = "2.13.1"
 val floggerVersion = "0.7.4"
-val gaxHttpJsonVersion = "0.96.0"
-val gaxVersion = "2.11.0"
-val googleApiCommonVersion = "2.1.3"
-val googleAuthVersion = "1.4.0"
-val googleCloudBigTableVersion = "2.5.3"
-val googleCloudDatastore = "0.93.4"
-val googleCloudMonitoringVersion = "3.2.4"
-val googleCloudSpannerVersion = "6.20.0"
-val googleCloudVersion = "2.4.0"
-val googleCommonsProtoVersion = "2.7.2"
-val googleHttpClientsVersion = "1.41.2"
-val googleIAMVersion = "1.2.1"
-val grpcVersion = "1.44.0"
-val jacksonVersion = "2.13.1"
-val opencensusContribGrpcMetricsVersion = "0.12.3"
+val gaxHttpJsonVersion = "0.101.0"
+val gaxVersion = "2.16.0"
+val googleApiCommonVersion = "2.1.5"
+val googleAuthVersion = "1.6.0"
+val googleCloudBigTableVersion = "2.6.2"
+val googleCloudDatastore = "0.95.0"
```

Comment:
Looks like beam overrides this to be `0.93.10`:
https://github.com/apache/beam/blob/release-2.40.0/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L683

Seems to be due to some explicit request: https://github.com/apache/beam/pull/17497#pullrequestreview-959120468